zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -781,6 +784,11 @@ private static void sendToKafka(Collection collection) throws IOException {
         sendMessage (collection, reviewedUris, "Reviewed-uris");
     }
 
+    // valid uris for published CMD versions of a dataset (edition) - /dataset/{datatsetId}/editions/{edition}/versions/{version}/metadata
+    protected static boolean isValidUris (String uri){
+        return uri.split("/").length > 6;
```

Comment:
I had some concerns about the (small) overhead of splitting the string and assigning them to a list just to count the occurrences. I looked for some guides o best ways to count a specific char in Java and there's an entire article on it…
https://www.baeldung.com/java-count-chars
Not really bothered as it's only a small operation so I'll still approve anyway but thought I'd give you the opportunity to decide for yourself.

Reply:
Thanks for pointing out, method name now updated with readable code using core library